direct-message-key.js

```diff
@@ -47,8 +47,14 @@ function EasyDirectMessageKey (ssbKeys) {
   }
 
   return function EasyDirectMessageKey (feedId) {
+    const dotIndex = feedId.indexOf('.')
+    const keys = {
+      public: feedId.substring(1, dotIndex > 0 ? dotIndex : feedId.length)
+      // prunes the @ + suffix
+    }
```

Comment:
NOTE @arj03 I modified this code to make it simpler/ easier to read (IMO)

tests pass so guess it's fine.